src/mixins/CopyToClipboardMixin.js

```diff
@@ -10,23 +10,23 @@ export default {
 	},
 
 	methods: {
-		async copyToClipboard(url) {
+		async copyToClipboard(content) {
 			// change to loading status
 			this.copyLoading = true
 
 			// copy link to clipboard
 			try {
-				await navigator.clipboard.writeText(url)
+				await navigator.clipboard.writeText(content)
 				this.copySuccess = true
 				this.copied = true
 
 				// Notify success
-				showSuccess(t('collectives', 'Link copied'))
+				showSuccess(t('text', 'Copied to the clipboard'))
 			} catch (error) {
 				this.copySuccess = false
 				this.copied = true
 				showError(
-					`<div>${t('collectives', 'Could not copy link to the clipboard:')}</div><div>${url}</div>`,
```

Comment:
If I remember correctly the url was in here on purpose. The idea was that if your browser cannot copy the url for you its still displaying it so you can copy it yourself.

I'm not sure how often people would need to do that. I remember that back in the days I could not copy share links because of my browser configuration and did not find a way around that.

However in this case I think it's different. I could open the link in a tab and copy the link from there - or I right click the link and select 'copy link' from the context menu.

So I'm fine with removing the url - just wanted to explain the context.